client/src/components/Navigation/Navigation.js

```diff
@@ -0,0 +1,36 @@
+import { Link } from 'react-router-dom'
+import styled from 'styled-components/macro'
+import ButtonHome from './Buttons/ButtonHome/ButtonHome'
+import ButtonVoting from './Buttons/ButtonVoting/ButtonVoting'
+
+const Navigation = () => {
+  return (
+    <Nav>
+      <NavLink to="/">
+        <ButtonHome />
+      </NavLink>
+      <NavLink to="/voting">
+        <ButtonVoting />
+      </NavLink>
+    </Nav>
+  )
```

Comment:
Your usage of NavLinks and Buttons is not semantic. As far as I know it is bad practice to put a block element in an inline element. But you could also use the react router NavLink that you could style as flexbox. We can speak about this if you like. :-)

Reply:
Changed it, thank you!